assets/router/deployment.yaml

```diff
@@ -4,6 +4,7 @@ kind: Deployment
 apiVersion: apps/v1
 # name and namespace are set at runtime.
 spec:
+  progressDeadlineSeconds: 120
```

Comment:
This might be too long, but 120 seems conservative and we can adjust if necessary.